Wireframe/css/style.css

```diff
@@ -0,0 +1,82 @@
+:root {
+  --backColor: #44209b;
+  --SbackColor: #f0f8ff;
+  --textColor: #000000;
+  --linkColor: rgb(55, 253, 5);
+}
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  box-sizing: border-box;
+}
+h3 {
+  color: var(--SbackColor);
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  margin: 0 30px 0 30px;
+  background-color: var(--SbackColor);
+}
+header {
+  background-color: var(--backColor);
+  color: var(--SbackColor);
+  text-align: center;
+  padding: 10px;
+}
+/* ---articles--- */
+header > h1:first-child {
+  color: var(--linkColor);
+}
+main {
+  margin-top: 10px;
+}
+main > div:nth-child(2) {
+  display: flex;
+  gap: 10px;
+  margin-top: 10px;
+}
+article {
+  padding: 0 10px;
+  background-color: var(--SbackColor);
+  box-shadow: 1px 1px 1px 1px var(--textColor);
+}
+.first__article {
+  padding: 5px 10px;
+}
+#reed__more {
```

Comment:
How could you style the read more links without using an ID? You'll want them to look similar, I think.